sklearn/ensemble/tests/test_forest.py

```diff
@@ -228,18 +231,20 @@ def check_importances(name, criterion, X, y):
         assert_less(np.abs(importances - importances_bis).mean(), 0.001)
```

Comment:
@ogrisel @jnothman All the builds fail with this error message at this line -

```py
AssertionError: 0.0067716825379038909 not less than 0.001
```

I'm not sure if this has anything to do with the dataset per se.

@glouppe @jmschrei is it safe to reduce the tolerance of this test to 0.01 to pass the test?